django_app/core/templates/core/base.html

```diff
@@ -280,6 +280,10 @@ <h2 class="govuk-visually-hidden">Support links</h2>
                                 <a class="govuk-footer__link"
                                    href="{% url 'accessibility_statement' %}">Accessibility statement</a>
                             </li>
+                            <li class="govuk-footer__inline-list-item hide-on-print">
+                                <a class="govuk-footer__link"
+                                   href="{% url 'help_and_support' %}">Help and Support</a>
```

Comment:
```suggestion
                                   href="{% url 'help_and_support' %}">Help and support</a>
```

Might be worth checking with Rebecca if this should also be lowercase